elixir/apps/web/lib/web/components/navigation_components.ex

```diff
@@ -308,4 +308,32 @@ defmodule Web.NavigationComponents do
     </div>
     """
   end
+
+  @doc """
+  Renders links based off our website path.
+
+  ## Examples
+
+    <.website_link href="/pricing>Pricing</.website_link>
+    <.website_link href="/kb/deploy/gateways" class="text-neutral-900">Deploy Gateway(s)</.website_link>
+    <.website_link href={~p"/contact/sales"}>Contact Sales</.website_link>
+  """
+  attr :href, :string, required: true
+  slot :inner_block, required: true
+  attr :rest, :global
+
+  # XXX: We could add automated testing to make sure these don't return 404s, but that will fail
```

Comment:
Please use TODO: or better create a GitHub issue for this, because it won't be trivial to implement this and nobody greps codebase by XXX right now

Reply:
Updated to remove. I don't think this will be a major issue and probably not worth investing the time to implement and maintain it